test/tests/misc/multiple-nl-systems/restart_problem.i

```diff
@@ -0,0 +1,78 @@
+[Mesh]
+  file = 'gold/starting_problem_out.e'
```

Comment:
cant you use restart_problem_out for the inital file?

Reply:
The issue I found is that when restarting, even if no calculation is done, is that it will add a 2nd timestep that is the same as the first. So when you go to compare input and output files one has a time of 1 and the other 2 and exodiff fails. That's why I have it read in starting_problem_out.e, and compare its output to restart_problem_out.e.

Reply:
I think start_time might only work for transient solvers? I tried using it but the test then errors because start_time is never used. I guess saying that it adds a 2nd timestep isn't accurate, but it adds a 2nd result. If there's a way to have a restart overwrite the previous output file instead of adding to it that might work. Otherwise the two output files might be necessary?

Reply:
there is a `time` parameter for steady

it should not be appending, you are not outputting to gold/starting_problem.e

Reply:
I tried messing around with the time parameter and I wasn't able to get it to quite work but I think I found a possible solution? I don't know if this is the right approach but adding execute_on = FINAL to the outputs block stops it from doing any execution after reading in the exodus file, so it should just read in the exodus file, copy over the solutions, and then output it right back out. Now when I compare its output against the file it reads in it matches. If that sounds like a reasonable solution then I can go ahead and push those changes.

Reply:
That sounds good
I suspect execute_on = initial is all you need to compare the old exodus with the just read then output exodus

Reply:
When I used execute_on = initial the output file was different from the input. I ended up using execute_on = final and pushed that change.